archinstall/lib/installer.py

```diff
@@ -48,8 +48,10 @@ def __init__(self, target, *, base_packages='base base-devel linux linux-firmwar
 			'base' : False,
 			'bootloader' : False
 		}
-
+		
 		self.base_packages = base_packages.split(' ') if type(base_packages) is str else base_packages
+		if hasUEFI() == False:
+			self.base_packages.pop(self.base_packages.index("efibootmgr"))
```

Comment:
Maybe efibootmgr shouldn't be in the base packages, instead we should maybe add it just like we do with the grub one? But this works and I can merge it :)

Reply:
sure, it would slim down the base packages. if you want I can change efibootmgr to be added like grub